src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -393,4 +395,49 @@ private boolean userHasPermission(Course course) {
         courseService.delete(id);
         return ResponseEntity.ok().headers(HeaderUtil.createEntityDeletionAlert(ENTITY_NAME, title)).build();
     }
+
+    /**
+     * GET  /courses/:courseId/results : Returns all results of the exercises of a course for the currently logged in user
+     *
+     * @param courseId the id of the course to get the results from
+     * @return the ResponseEntity with status 200 (OK) and with body the exercise, or with status 404 (Not Found)
+     */
+    @GetMapping(value = "/courses/{courseId}/results")
+    @PreAuthorize("hasAnyRole('USER', 'TA', 'INSTRUCTOR', 'ADMIN')")
+    @Timed
```

Comment:
after the latest changes on the server @Times is not necessary any more. Please remove it.